pkg/openevec/config.go

```diff
@@ -21,6 +22,12 @@ type EServerConfig struct {
 	Tag    string       `mapstructure:"tag" cobraflag:"eserver-tag"`
 	IP     string       `mapstructure:"ip"`
 	Images ImagesConfig `mapstructure:"images"`
+	EVEIP  string       `mapstructure:"eve-ip"`
```

Comment:
is it used anywhere?

Reply:
Needed for backward compatibility with escript tests, some of them use it, plus that's how we understand which IP has been assigned to EVE :)